101-javaee-like-getting-started/src/main/java/io/quarkus/qe/core/GreetingHealthCheck.java

```diff
@@ -1,12 +1,13 @@
 package io.quarkus.qe.core;
 
-import org.eclipse.microprofile.health.Health;
```

Comment:
`Health` annotation got removed in Micro Profile 4 ?

Reply:
The annotation `@Health` was deprecated in version 2.2 (Quarkus 1.13.1.Final) and finally dropped in version 3 (Quarkus 2.x).
In order to inject a custom health check in liveness and/or readiness, you need to use the annotations `@Liveness` and `@Readiness`.

Reply:
I've changed the @Health to @Readiness and added a new test case to cover @HealthGroup.